docs/opensauced-guides/oscr-score-guide/oscr-introduction.md

```diff
@@ -0,0 +1,103 @@
+---
+id: oscr-guide
+title: "Guide to Improving Your OSCR"
+sidebar_label: How to Improve Your OSCR Score
+keywords: 
+- "contributors"
+- "guides"
+- "how to contribute to open source"
+- "OSCR score"
+- "Make an impact in open source"
+- "Open Source Contributor Rating"
+- "Open Source Score"
+---
+
+Open Source Contributor Rating ([OSCR](../../welcome/glossary.md#oscr)) is a metric that measures participation in open source projects. It is a holistic metric that considers the quality of contributions across many areas over a 90-day period. It is scored out of 300.
+
+As OSCR is a simple way to measure your impact in open source and can act as a positive signal of your value to potential employers, you may want to improve your score. This guide will help you understand how to do so.
```

Comment:
Let's break this down differently to prevent a run-on sentence:

The OSCR is one way to measure your impact in open source and can act as a positive signal of your value. However, you may want to improve your score. This guide will help you understand how to do so.